packages/ui/src/proposals/components/ProposalDetails/ProposalDetails.tsx

```diff
@@ -72,6 +75,24 @@ export const ProposalDetails = ({ proposalDetails }: Props) => {
   const detailsRenderStructure = useMemo(() => getDetailsRenderStructure(proposalDetails), [proposalDetails])
 
   const additionalDetails = useMemo(() => {
+    if (gracePeriod) {
+      return [
```

Comment:
I missed that on the first review but if this logic is in this `useMemo`: `setReferralCut`, `fundingRequest`, and `updateWorkingGroupBudget` will be missing there "Amount" statistics. E.g [`setReferralCut`](https://pioneer-2-storybook-git-fork-vrrayz-issue-4319-joystream.vercel.app/?path=/story/pages-proposals-proposalpreview--set-referral-cut).

Instead either "Gracing Period" or "Exact Execution Block" should be append to the statistics (just like `additionalDetails` already is).